src/postscriptlight.c

```diff
@@ -3158,6 +3158,10 @@ static int psl_init_fonts (struct PSL_CTRL *PSL) {
 		fclose (in);
 		PSL->internal.N_FONTS = i;
 	}
+	else {
+		PSL_message (PSL, PSL_MSG_LONG_VERBOSE, "No PSL_custom_fonts.txt found\n");
```

Comment:
I'm afraid this can do more harm than good. ``PSL_custom_fonts.txt`` is not a file users normally have or should have. So seeing a warning that something was not found makes think in a somewhat broken installation.

Reply:
it is not a warning, it is information under -Vl.